com.unity.render-pipelines.universal/Runtime/Passes/MainLightShadowCasterPass.cs

```diff
@@ -214,10 +221,16 @@ void SetupMainLightShadowReceiverConstants(CommandBuffer cmd, VisibleLight shado
             float invShadowAtlasHeight = 1.0f / m_ShadowmapHeight;
             float invHalfShadowAtlasWidth = 0.5f * invShadowAtlasWidth;
             float invHalfShadowAtlasHeight = 0.5f * invShadowAtlasHeight;
+            float softShadowsProp = softShadows ? 1.0f : 0.0f;
+
+            Vector4 lastCascade = m_CascadeSplitDistances[m_ShadowCasterCascadesCount - 1];
+            float lastCascadeRadiusSq = lastCascade.w * lastCascade.w;
+            ShadowUtils.GetScaleAndBiasForLinearDistanceFade(lastCascadeRadiusSq, m_CascadeBorder, out float shadowFadeScale, out float shadowFadeBias);
 
             cmd.SetGlobalTexture(m_MainLightShadowmap.id, m_MainLightShadowmapTexture);
             cmd.SetGlobalMatrixArray(MainLightShadowConstantBuffer._WorldToShadow, m_MainLightShadowMatrices);
-            ShadowUtils.SetupShadowReceiverConstantBuffer(cmd, m_MainLightShadowParams);
```

Comment:
This is no longer needed, as additional light calculation no longer uses main light